content/docs/data/Import.md

```diff
@@ -0,0 +1,319 @@
+		<img class="w10 mb1" src="https://i.imgur.com/qXDGYSX.png" />
+		Dashboard
+	</a>
+	<a class="bg-white shadow-2 box-shadow-hover shadow-2-hover  br4 db flex flex-column justify-between items-center middarkgrey pa2 pt5 pb5 tdn tc" style="box-shadow: 0 0 5px rgba(0,0,0,.02), 0 5px 22px -8px rgba(0,0,0,.1);    word-break: normal;cursor: pointer; padding: 2rem; height: 120px;width:120px;" href="#importing-through-abc-cli">
+    		<img class="w10 mb1" src="https://user-images.githubusercontent.com/4047597/29240054-14e0e19a-7f7b-11e7-898b-ba6bad756b1d.png" />
+    		ABC CLI
+    </a>
+    <a class="bg-white shadow-2 box-shadow-hover shadow-2-hover  br4 db flex flex-column justify-between items-center middarkgrey pa2 pt5 pb5 tdn tc" style="box-shadow: 0 0 5px rgba(0,0,0,.02), 0 5px 22px -8px rgba(0,0,0,.1);    word-break: normal;cursor: pointer; padding: 2rem; height: 120px;width:120px;" href="#rest-api">
+        		<img class="w10 mb1" src="https://i.imgur.com/nKKQLXb.png" />
+        		Rest API
+     </a>
+	<a class="bg-white shadow-2 box-shadow-hover shadow-2-hover  br4 db flex flex-column justify-between items-center middarkgrey pa2 pt5 pb5 tdn tc" style="box-shadow: 0 0 5px rgba(0,0,0,.02), 0 5px 22px -8px rgba(0,0,0,.1);    word-break: normal;cursor: pointer; padding: 2rem; height: 120px;width:120px;" href="#zapier">
+		<img class="w10 mb1" src="https://i.imgur.com/Z9ahZbM.png" />
+		Zapier
+	</a>
+</div>
+
+---
+
+##Importing through Dashboard
+
+Dashboard is a WYSIWYG GUI for adding, modifying and viewing your appbase.io app's data.
+
+Data is stored as JSON documents. You can read more about the data schema [here](/docs/data/Model).
+
+In this tutorial we will review how to import data from a data file using the appbase.io [dashboard](https://dashboard.appbase.io/).
+
+### Dataset
+
+We’ll use an example from an [actors dataset](https://github.com/algolia/datasets/blob/master/movies/actors.json). Feel free to use your own dataset instead.
+
+We use JSON format, but CSV is also acceptable.
+
+**Example of a JSON file**
+
+```json
+[
+	{
+		"firstname": "Jimmie",
+		"lastname": "Barninger"
+	},
+	{
+		"firstname": "Warren",
+		"lastname": "Speach"
+	}
+]
+```
+
+**Example of a CSV file**
+
+```csv
+firstname,lastname
+Jimmie,Barninger
+Warren,Speach
+```
+
+### Open the appbase.io Dashboard
+
+Head over to [appbase.io dashboard](https://dashboard.appbase.io) and login with your credentials or create an account if you haven't one.
+
+### Creating a new app
+
+As we are using an actors dataset, let’s name our new app `actors`.
+
+![](https://i.imgur.com/v4h2JuY.png)
+
+### Upload the file
+
+Now that the app has been created, we can import the data file as shown in the GIF.
+
+![](https://www.dropbox.com/s/05do1q1wvp7t13t/short_upload.gif?raw=1)
+
+---
+
+##Importing through ABC CLI
```

Comment:
@ShahAnuj2610 you're missing adaptors. See here for a list of all: https://github.com/appbaseio/abc/tree/dev/docs/importer/adaptors.